File: example/kotlinlib/testing/3-integration-suite/qux/test/src/qux/QuxTests.kt
package qux

import io.kotest.core.spec.style.FunSpec
import io.kotest.matchers.string.shouldEndWith
import io.kotest.matchers.string.shouldStartWith

class QuxTests :
    FunSpec({

        test("hello") {
            val result = Qux.hello()
            result shouldStartWith "Hello"
        }

        test("world") {
            val result = Qux.hello()
            result shouldEndWith "World"
        }
    })

File: integration/feature/test-quick/resources/qux/test/src/qux/QuxTests.scala
package qux

import utest._

object QuxTests extends TestSuite {
  val tests = Tests {
    test("hello") {
      assert(Qux.hello() == "Hello")
    }
    test("world") {
      assert(Qux.world() == "World")
    }
  }
}
